client/blocks/importer/components/importer-drag/config.tsx

```diff
@@ -52,23 +14,67 @@ export function getImportDragConfig( importer: Importer, supportLinkModal?: bool
 			supportLink: <SupportLink importer={ importer } supportLinkModal={ supportLinkModal } />,
 		},
 	};
+	const translateConfig: { [ key: string ]: Partial< ImporterConfig > } = {
+		blogger: {
+			description: translate(
+				'Import posts, pages, comments, tags, and images from a %(importerName)s export file.',
+				options
+			),
+			uploadDescription: translate(
+				'A %(importerName)s export is ' +
+					'an XML file with your page and post content, or a zip archive ' +
+					'containing several XML files. ' +
+					'{{supportLink/}}',
+				options
+			),
+		},
+		medium: {
+			description: translate(
+				'Import your posts, tags, images, and videos from your %(importerName)s export file',
```

Comment:
Do we know why this sentence lacks a period (.), but the other descriptions do not?